src/Data/Recruit.ChangeScripts/010_vacancy_backfill_closureReason.js

```diff
@@ -3,13 +3,17 @@
     const autoClosure = "Auto";
     const manualClosure = "Manual";
 
-    function setClosureReason(vac, closureReason) {
+    function setClosureReason(vac) {
         print(`updating ${vac.vacancyReference}`);
-        db.vacancies.updateOne({ "_id": vac._id }, { $set: { "closureReason": closureReason} });
+        db.vacancies.updateOne({ "_id": vac._id }, { $set: { "closureReason": vac.closureReason} });
     }
 
     var closedVacancies = db.vacancies.aggregate([
-        { $match : {"status" : "Closed", "closureReason": {$exists: 0} }},
+        { $match : {"status" : "Closed", 
+            $or : [ 
```

Comment:
I think it will be simpler to avoid any check on closureReason field, as we need to update all the closed vacancies.

Reply:
I think we still need this check as we'd be overwriting any future withdrawn vacancies every time we deploy.